lib/sidekiq/form526_backup_submission_process/processor.rb

```diff
@@ -428,29 +430,39 @@ def convert_doc_to_pdf(doc, klass)
         end
       end
 
-      def choose_provider(headers, breakered: true)
+      # 82245 - Adding provider to method. this should be removed when toxic exposure flipper is removed
+      def choose_provider(headers, provider, breakered: true)
         ApiProviderFactory.call(
           type: ApiProviderFactory::FACTORIES[:generate_pdf],
-          # let Flipper - the feature toggle - choose which provider
-          provider: nil,
+          provider:,
           # this sends the auth headers and if we want the "breakered" or "non-breakered" version
-          options: { auth_headers: headers, breakered: },
+          options: { auth_headers: headers, breakered:, icn: user_account.icn },
```

Comment:
Could probably utilize the new `user_account` helper defined below to replace the `OpenStruct` on L440.